features/step_definitions/logging.rb

```diff
@@ -386,3 +387,128 @@
     | component=eventrouter |
   })
 end
+
+Given /^I generate certs for the#{OPT_QUOTED} receiver(?: in the#{OPT_QUOTED} project)?$/ do | receiver_name, project_name |
+  script_file = "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/logforwarding/cert_generation.sh"
+  project_name ||= "openshift-logging"
+  step %Q/I download a file from "#{script_file}"/
+  shell_cmd = "sh cert_generation.sh $(pwd) #{project_name} #{receiver_name}"
+  system(shell_cmd)
+end
+
+Given /^fluentd receiver is deployed as (secure|insecure)(?: in the#{OPT_QUOTED} project)?$/ do | security, project_name |
+  project_name ||= "openshift-logging"
+
+  org_user = user
+  project(project_name)
+  step %Q/I ensures "fluentdserver" service_account is deleted from the "<%= project.name %>" project after scenario/
+  step %Q/I ensures "fluentdserver" config_map is deleted from the "<%= project.name %>" project after scenario/
+  step %Q/I ensures "fluentdserver" deployment is deleted from the "<%= project.name %>" project after scenario/
+  step %Q/I ensures "fluentdserver" service is deleted from the "<%= project.name %>" project after scenario/
```

Comment:
@QiaolingTang I see duplication of these 4 steps, perhaps we can merged them into a single step

Reply:
Thanks @pruan-rht , I added a new single step to remove these resources, could you please review this PR again? If there is no other concerns, could you please help to merge it ?

Reply:
@QiaolingTang I don't see this unified step you mentioned...can you pls point me to it?

Reply:
@pruan-rht  https://github.com/openshift/verification-tests/pull/545/files#diff-0f0202f69f619cc471255d933a81861cR427-R455